mllib/src/main/scala/org/apache/spark/ml/Pipeline.scala

```diff
@@ -100,6 +101,15 @@ class Pipeline(override val uid: String) extends Estimator[PipelineModel] {
   /** @group getParam */
   def getStages: Array[PipelineStage] = $(stages).clone()
 
+  /** @group setParam */
```

Comment:
Probably worth it to include some Javadoc which explains how setSeed works (e.g. it seems to set the seed on all elements which haveSeed but are not set but a user might assume it overwrites the previous values).